react-ui/src/components/addOrder/OrderReferralRequest.js

```diff
@@ -0,0 +1,142 @@
+import React, {useContext, useEffect, useState} from 'react';
+import {
+    Column,
+    DatePicker,
+    DatePickerInput,
+    Grid,
+    Select,
+    SelectItem,
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableHeader,
+    TableRow,
+    TextInput
+} from "@carbon/react";
+import {UserInformationContext} from "../layout/Layout";
+
+
+const header = [{key: 'reason', header: 'Reason for Referral*'}, {key: 'referrer', header: 'Referrer'}, {
```

Comment:
headers will need to be message keys for translation

Reply:
Oh yeah, Let me start mapping the text to keys for translation.